doc/whats_new.rst

```diff
@@ -144,6 +144,18 @@ Enhancements
    - The :func: `ignore_warnings` now accept a category argument to ignore only
      the warnings of a specified type. By `Thierry Guillemot`_.
 
+   - The `grid_scores_` attribute of :class:`model_selection.GridSearchCV` and
```

Comment:
The removal of `grid_scores_` should be covered in API changes; it is not an enhancement.